rcl/include/rcl/time.h

```diff
@@ -349,6 +454,18 @@ rcl_clock_get_now(rcl_clock_t * clock, rcl_time_point_value_t * time_point_value
  * such that the time source will report the set value instead of falling
  * back to system time.
  *
+ * This function is not thread-safe with `rcl_clock_add_jump_callback`,
+ * nor `rcl_clock_remove_jump_callback` functions when used on the same
+ * clock object.
+ *
+ * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | No
+ * Thread-Safe        | No
+ * Uses Atomics       | No
+ * Lock-Free          | Yes
```

Comment:
Should we mention that this only covers the function itself? The callbacks invoked by this function could do anything.

Same for `rcl_disable_ros_time_override` and `rcl_set_ros_time_override`.

Reply:
Good point. Done in 03d4a769cf9515c6101a32b27c381af9322d6706.